public/app/features/dashboard/components/DashNav/IirisMaintenanceModal.tsx

```diff
@@ -803,8 +803,8 @@ export class IirisMaintenanceModal extends PureComponent<Props, State> {
   populateMinuteSelector = (minuteInputObject: any, minuteValue: number) => {
     for (let i = 0; i < 60; i++) {
       minuteInputObject.options.push({
-        text: i < 10 ? '0' + i : '' + i,
-        value: i < 10 ? '0' + i : '' + i,
+        text: '' + i,
```

Comment:
Line 810 also has the same logic and tries to find minutes with findIndex => probably fails if the logic differs

Reply:
In commit 73ce13d33f4167f9790a54a53e2121c3dee1cddf.